stats/opentelemetry/server_metrics.go

```diff
@@ -66,6 +68,15 @@ func (h *serverStatsHandler) initializeMetrics() {
 	rm.registerMetrics(metrics, meter)
 }
 
+func (h *serverStatsHandler) initializeTracing() {
+	if !isTracingDisabled(h.options.TraceOptions) {
+		return
+	}
+
+	otel.SetTextMapPropagator(h.options.TraceOptions.TextMapPropagator)
+	otel.SetTracerProvider(h.options.TraceOptions.TracerProvider)
```

Comment:
IIUC, these are global settings.  I don't think that's what we want...  Only the user should ever be calling these kinds of function, not our library.

Reply:
Ping?

Reply:
@dfawley you're right, ideally user should set these variables. I have updated the PR, kindly check.